addons/survey/tests/test_survey.py

```diff
@@ -39,11 +39,11 @@ def setUp(self):
         self.UserInput = self.env['survey.user_input']
         self.UserInputLine = self.env['survey.user_input_line']
 
-        self.survey1 = self.Survey.sudo(self.survey_manager).create({'title': "S0", 'page_ids': [(0, 0, {'title': "P0"})]})
+        self.survey1 = self.Survey.sudo(self.survey_manager.id).create({'title': "S0", 'page_ids': [(0, 0, {'title': "P0"})]})
         self.page1 = self.survey1.page_ids[0]
 
     def test_00_create_minimal_survey(self):
-        question = self.Question.sudo(self.survey_manager).create({'page_id': self.page1.id, 'question': 'Q0'})
+        question = self.Question.sudo(self.survey_manager.id).create({'page_id': self.page1.id, 'question': 'Q0'})
         self.assertEqual(self.survey1.title, "S0", msg="Title of the survey is somehow modified.")
```

Comment:
Improve test fail message like :confetti_ball: 
```
self.assertEqual(self.survey1.title, "S0", msg="Title of the survey shold be SO instead of %s."  % (self.survey1.title))
```